src/main/java/frc/robot/commands/TurnAngle.java

```diff
@@ -0,0 +1,73 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+import edu.wpi.first.wpilibj2.command.PIDCommand;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj.interfaces.Gyro;
+
+public class TurnAngle extends CommandBase {
+  /**
+   * Creates a new TurnAngle.
+   */
+
+  private Drivetrain drivetrain;
+  private Gyro gyro;
```

Comment:
The 'navx' I was referring to is a gyro, I believe the class name is AHRS. You might need to install it as a 3rd party dependency.

Reply:
Ok, I didn't see this comment when I saw the others, I will do this in another pull request.